MacOSX-Framework

```diff
@@ -87,8 +87,8 @@ then
 else
  ARCHES64='-arch x86_64'
  #We "know" that 10.4 and earlier do not support 64bit
- OLD_SDK64=`ls  $SDK_PATH|egrep -v "10.[0-4]"|head -1`
- NEW_SDK64=`ls -r $SDK_PATH|egrep -v "10.[0-4][^0-9]" | head -1`
+ OLD_SDK64=`ls  $SDK_PATH|grep -Ev "10.[0-4]"|head -1`
+ NEW_SDK64=`ls -r $SDK_PATH|grep -Ev "10.[0-4][^0-9]" | head -1`
```

Comment:
you don't need to use `-E` here

Reply:
I didn't want to change the script more than I had to.

Reply:
Strictly speaking, `.` should be escaped here as well. This, and dropping `-E` are fine within this PR IMO.

We have more `egrep`s in this file:
```
.github/workflows/hacktoberfest-accepted.yml:            egrep -i "^Close[sd]? " | sort | uniq | tee log
.github/workflows/hacktoberfest-accepted.yml:          egrep -o "#([0-9]+)" log | cut -d# -f2 | sort | uniq | xargs -t -n1 -I{} \
.github/workflows/hacktoberfest-accepted.yml:            egrep -o '^([0-9]+):[0-9]{4}-(09-30T|10-|11-01T)' | cut -d: -f1 | sort | uniq | xargs -t -n1 -I {} \
.github/workflows/hacktoberfest-accepted.yml:          egrep -o "github.com/(.+)/(.+)/pull/([0-9]+)" log | sort | uniq | xargs -t -n1 -I{} \
.github/workflows/hacktoberfest-accepted.yml:            egrep -o '^([0-9]+):[0-9]{4}-(09-30T|10-|11-01T)' | cut -d: -f1 | sort | uniq | xargs -t -n1 -I {} \
```

Reply:
Addressed your comments in the fixed up commit.